CMakeLists.txt

```diff
@@ -100,7 +100,8 @@ if(ENABLE_FOLLY)
   else()
     set(FOLLY_LIBRARIES "")
     add_definitions("-DHAVE_FOLLY")
-    list(APPEND Folly_LIBRARIES Folly::folly ${Glog_LIBRARIES})
+    find_package(gflags CONFIG REQUIRED)
+    list(APPEND Folly_LIBRARIES Folly::folly)
```

Comment:
cmake variables are case sensitive, did you mean to add it to the `FOLLY_LIBRARIES`?